.github/workflows/release-bundle-to-charmhub.yaml

```diff
@@ -1,40 +0,0 @@
-name: Publish bundle to Charmhub
-on:
-  push:
```

Comment:
For some reason, this still exists as an empty file https://github.com/canonical/bundle-kubeflow/blob/f00d234259d92ed93050ae443c14a090929d150b/.github/workflows/release-bundle-to-charmhub.yaml